book/website/community-handbook/presenting.md

```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
+### Planning a Talk
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
+
+### Preparing Slides
+Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.
+
```

Comment:
Add here - You can also see all our previous talks on Zenodo by searching for _The Turing Way_. Here is a [link to _The Turing Way_ Community on Zenodo](https://zenodo.org/communities/the-turing-way/?page=1&size=20).

Reply:
Are you able to add the link to the promotion pack Google folder?

Reply:
An alternative could be to not use Google Drive at all and instead put the promotion pack on zenodo to be downloaded

Reply:
> unlike the pdfs uploaded to zenodo.

I take your point RE this specific case. I want to make a general point about the Zenodo. It is a failure point if people are only uploading PDFs to Zenodo. It can handle multiple file formats and people should be sharing **both** in the Zenodo record (PDF + whatever they created the PDF from). Because then the pptx or whatever would also be clonable and editable.